lms/static/sass/base/_grid-settings.scss

```diff
@@ -1,16 +1,24 @@
-@import "neat/neat-helpers"; // or "neat-helpers" when in Rails
```

Comment:
NOTE: cleaning up Sass comment syntax here while I'm in the file (these ``//`` comments will not render within our production CSS).